File: discover_ruby/github_most_popular.rb
require "HTTPClient"
require "net/http"
require "uri"

extheaders = {
  'User-Agent' => 'Holberton_School',
  'Authorization' => 'token 3e0df28fcee55ed9d9bfbfda999ac5428e1ab96c'
}


clnt = HTTPClient.new

puts clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc')

File: python_intro_2/github_most_popular.py
import urllib2

url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'

"""Prints standard url request to standard output"""
request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token 3e0df28fcee55ed9d9bfbfda999ac5428e1ab96c'
}

req = urllib2.Request(url, headers=request_headers)
response = urllib2.urlopen(req)
the_page = response.read()
print the_page
